docs/utils.py

```
@@ -126,3 +139,34 @@
+	sanitized_url = urlunparse((
+		parsed_url.scheme,
+		parsed_url.netloc,
+		sanitized_path,
+		parsed_url.params,
+		sanitized_query,
+		parsed_url.fragment
+	))
+	return sanitized_url
+
+
+def sanitize_intersphinx_mapping(mapping):
```

Ruff (0.8.2)

Missing return type annotation for public function `sanitize_intersphinx_mapping`

(ANN201)